portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -153,11 +160,10 @@ private int getIntParameter(HttpServletRequest request, String userValue, int de
      *         was not specified by the client
      */
     private Boolean getBooleanParameter(HttpServletRequest request, String userValue, boolean defaultValue) {
-        Boolean value = defaultValue;
-        value = Boolean.parseBoolean(request.getParameter(userValue));
-        if (value == null) {
+        String toParse = request.getParameter(userValue);
+        if (toParse == null || !("true".equals(toParse.toLowerCase()))) {
```

Comment:
How is the parameter supposed to represent properly sending a value of "false"?

Consider the case of the parameter being "false" (which is a perfectly valid value) and the default being true. (since this week is TDD, perhaps add some tests to cover this case and ensure desired behavior